v10.2/bingo.html

```diff
@@ -57,7 +57,7 @@ <h2>About</h2>
                                 must not drink it, and you must still have it in your inventory at the time you finish
                                 getting all 5 objectives. </li>
                             <li>When an item may be marked as completed: <br>
-                                Song: When the text says "You have learned the..." <br>
+                                Song: When the dialog box that says "You have learned the ..." completes.<br>
```

Comment:
This wording is still ambiguous in my opinion. What does complete mean? Can be interpreted as
- Last letter/punctuation on screen
- Blue box on screen
- Input to clear the text box

Also there shouldn't be a space before the ellipsis, because it's at the end of the sentence.

Gonna leave this PR open, but taking the discussion back to Discord.